public/sass/partials/_app-support.scss

```diff
@@ -0,0 +1,116 @@
+.app-support {
+  position: relative;
+  @media only screen and (min-width: 550px) {
```

Comment:
consider using a breakpoint mixin

https://responsivedesign.is/develop/getting-started-with-sass